### PR TITLE
refactor: foundations

### PR DESCRIPTION
**Pull request by nmerget**
## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Refactoring (fix on existing components or architectural decisions)
-   [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Review by mfranzke — DISMISSED** [review]

File: packages/components/src/components/accordion-item/accordion-item.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @use "@db-ui/foundations/build/scss/variables";
 @use "@db-ui/foundations/build/scss/helpers/functions";
-@use "@db-ui/foundations/build/scss/helpers/component";
-@use "@db-ui/foundations/build/scss/color/color-variants";
+@use "../../styles/component";
+@use "@db-ui/foundations/build/scss/color/variants";
```
Comment (mfranzke) [inline packages/components/src/components/accordion-item/accordion-item.scss:4]: @nmerget let's merge `variants` and `code-colors` here into `index.scss`, as we would elsewhere miss the general namespace of `colors` within the upfollowing references (like e.g. `@include variants.bg-transparent-interactive(false);` in this file, which doesn't include the indicator or `color` anymore …).

Comment (nmerget) [reply packages/components/src/components/accordion-item/accordion-item.scss]: Add `_index.scss` files to reduce complexity :)

File: packages/foundations/scss/_init.scss [hunk 2]
```diff
@@ -17,11 +27,23 @@ body {
...
 // Box sizing
 html {
 	box-sizing: var(--db-box-sizing, border-box);
 }
 
+// global
+* {
+	@extend %db-font-size-md;
+}
+
```
Comment (mfranzke) [inline packages/foundations/scss/_init.scss:46]: We need to reevaluate these declarations, as those are defined quite globally.

**Review by mfranzke — APPROVED** [review]

File: packages/foundations/scss/_init.scss (hunk 2 above)
Comment (mfranzke) [reply packages/foundations/scss/_init.scss:46]: created an issue for that: https://github.com/db-ui/mono/issues/1681

**Pull request merged**